### PR TITLE
Feat: Update Resume Check

### PR DESCRIPTION
**Pull request by tharshita**
# Notion Task

https://www.notion.so/Update-checking-button-afda93ce84a041a98c0ce55e489e36ef

# Changes Made In This PR
- Modified checking section in About component
- linked offerings to go to https://www.careersocius.com/resume-check (does not exist yet)
# Screenshots

_for UI changes, show a before and after shot_\
**Before:**
Screenshot 2022-03-19 at 5 10 58 PM

**After:**
Screenshot 2022-03-19 at 5 12 13 PM

### CODE REVIEW

File: src/i18n/locales/en.json [hunk 1]
```diff
@@ -540,5 +540,12 @@
     "heading1": "You're previewing version",
     "heading2": "created at",
     "text": "Revert to this version?"
+  },
+  "aboutCheck": {
+    "resumecheck": "Get your resume checked!",
+    "cs": "CareerSocius",
+    "description": "is made up of a community of personal branding partners who can help you improve on your resume. Get matched with a professional in your related field/ industry to provide you with pointers to improve your resume!",
+    "offering": "View offerings",
```
Comment (millasml) [inline src/i18n/locales/en.json:548]: capitalize the "O"

**Review by millasml — CHANGES_REQUESTED** [review]
minor!

File: src/i18n/locales/en.json (hunk 1 above)
Comment (tharshita) [reply src/i18n/locales/en.json]: okies done!

**Review by millasml — APPROVED** [review]
yay!

**Pull request merged**